### PR TITLE
Pedro/fixes numpy changes

### PR DESCRIPTION
**Pull request by pedrocamargo**
Some tests had broken because of changes in behavior for NumPy

### CODE REVIEW

**Comment by pedrocamargo** [comment]
@djfrancesco I also formatted a few strings since I was at it...

File: aequilibrae/distribution/gravity_calibration.py [hunk 1]
```diff
@@ -218,15 +218,15 @@ def __check_inputs(self):
         mats = [(self.matrix, "Observed matrix"), (self.impedance, "Impedance matrix")]
         for matrix, title in mats:
             if matrix.matrix_view is None:
-                raise ValueError(title + " needs to be set for computation")
+                raise ValueError("{title} needs to be set for computation")
             else:
                 if len(matrix.matrix_view.shape[:]) > 2:
-                    raise ValueError(title + "' computational view needs to be set for a single matrix core")
+                    raise ValueError("{title}' computational view needs to be set for a single matrix core")
```
Comment (djfrancesco) [inline aequilibrae/distribution/gravity_calibration.py:224]: f prefix seems to be also missing for f-string? And maybe it is intentional but there is an isolated quote ' ?

File: aequilibrae/distribution/gravity_calibration.py [hunk 2]
```diff
@@ -218,15 +218,15 @@ def __check_inputs(self):
         mats = [(self.matrix, "Observed matrix"), (self.impedance, "Impedance matrix")]
         for matrix, title in mats:
             if matrix.matrix_view is None:
-                raise ValueError(title + " needs to be set for computation")
+                raise ValueError("{title} needs to be set for computation")
             else:
                 if len(matrix.matrix_view.shape[:]) > 2:
```
Comment (djfrancesco) [inline aequilibrae/distribution/gravity_calibration.py:223]: Maybe we could use something like `matrix.matrix_view.ndim > 2` ?

File: aequilibrae/matrix/aequilibrae_matrix.py [hunk 3]
```diff
@@ -1021,8 +1021,7 @@ def nan_to_num(self):
             raise NotImplementedError("This operation does not make sense for OMX matrices")
 
         if np.issubdtype(self.dtype, np.floating) and self.matrix_view is not None:
```
Comment (djfrancesco) [inline aequilibrae/matrix/aequilibrae_matrix.py:1023]: Sometimes it helps for readability to put parenthesis in conditions : 

```python
if np.issubdtype(self.dtype, np.floating) and (self.matrix_view is not None):
```

**Review by djfrancesco — COMMENTED** [review]
Hi Pedro, I am not familiar with the source code yet, so I do not understand precisely what is done here. I still left a few comments and hope this helps

File: aequilibrae/distribution/gravity_calibration.py (hunk 1 above)
Comment (jamiecook) [reply aequilibrae/distribution/gravity_calibration.py:224]: good catch - fixed.

File: aequilibrae/matrix/aequilibrae_matrix.py (hunk 3 above)
Comment (jamiecook) [reply aequilibrae/matrix/aequilibrae_matrix.py:1023]: I believe that tools like black will remove unnecessary parentheses... I don't particularly find either better or worse.

File: aequilibrae/distribution/gravity_calibration.py [hunk 4]
```diff
@@ -218,15 +218,15 @@ def __check_inputs(self):
         mats = [(self.matrix, "Observed matrix"), (self.impedance, "Impedance matrix")]
         for matrix, title in mats:
             if matrix.matrix_view is None:
-                raise ValueError(title + " needs to be set for computation")
+                raise ValueError("{title} needs to be set for computation")
```
[the remark this answers is not in the document]
Comment (jamiecook) [reply aequilibrae/distribution/gravity_calibration.py:221]: fixed

File: aequilibrae/distribution/gravity_calibration.py (hunk 2 above)
Comment (jamiecook) [reply aequilibrae/distribution/gravity_calibration.py:223]: agreed

**Review by jamiecook — COMMENTED** [review]
address issues from code-review

**Review by jamiecook — APPROVED** [review]

**Pull request merged**